homeassistant/components/sensor/glances.py

```diff
@@ -170,12 +162,11 @@ def __init__(self, resource):
         self._resource = resource
         self.data = dict()
 
-    @Throttle(MIN_TIME_BETWEEN_UPDATES)
```

Comment:
You will still need the Throttle. Each entity will call `update` and thus if you have 5 entities, you will update 5 times!